app/components/ak-button/index.scss

```diff
@@ -0,0 +1,29 @@
+.btn {
+  height: 2.5rem;
+  padding: 0.57em 0.85em;
+  border: none;
+  border-radius: var(--button-border-radius);
+  font-size: 1rem;
+  display: flex;
+  align-items: center;
+  flex-direction: row;
+  justify-content: center;
+  column-gap: 0.5em;
+  line-height: 2.5rem;
+  cursor: pointer;
+
+  span {
```

Comment:
use class there no direct reference to tag

Reply:
yes, earlier it was a problem, now it is not anymore required!